client/my-sites/customer-home/cards/launchpad/blog-flow.tsx

```diff
@@ -0,0 +1,112 @@
+import { useLaunchpad } from '@automattic/data-stores';
+import { Task } from '@automattic/launchpad';
+import { isMobile } from '@automattic/viewport';
+import { addQueryArgs } from '@wordpress/url';
+import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import CustomerHomeLaunchpad from '.';
+
+const checklistSlug = 'blog-flow';
+
+interface LaunchpadKeepBuildingProps {
+	siteSlug: string | null;
+}
+
+const LaunchpadBlogFlow = ( { siteSlug }: LaunchpadKeepBuildingProps ): JSX.Element => {
+	const {
+		data: { checklist },
+	} = useLaunchpad( siteSlug, checklistSlug );
+
+	const numberOfSteps = checklist?.length || 0;
+	const completedSteps = ( checklist?.filter( ( task: Task ) => task.completed ) || [] ).length;
+	const tasklistCompleted = completedSteps === numberOfSteps;
```

Comment:
This will be true when both values are `0`. I think we should add an explicit check for that to avoid weird bugs.
```suggestion
	const tasklistCompleted = numberOfSteps > 0 && completedSteps === numberOfSteps;
```